City-of-Helsinki/youth-membership-ui
# YM-406 | Generate new types

## Description
Changed `.graphql` files to match new spec. Generated new types and changed them across the application.




[YM-406](https://helsinkisolutionoffice.atlassian.net/browse/YM-406)

## How Has This Been Tested?

Manual testing. Application compiles without errors and and user is able to login, enter birthdate and enter creation form. Submitting form is not yet possible.

## Manual Testing Instructions for Reviewers

1) Open localhost
2) Enter birthdate + login
3) You should be redirected to creation form without errors.
